hr_skill/tests/test_hr_skill.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestHrSkill(TransactionCase):
+
+    def test_name_get(self):
+        hr_skill_model = self.env['hr.skill']
+        soft_skill_id = hr_skill_model.create({'name': 'Soft skill'})
+        leadership_skill_id = hr_skill_model.create({
```

Comment:
You could remove the suffix _id for `soft_skill_id` and `leadership_skill_id` as those 2 variables represent an instance of object.